src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -51,6 +54,14 @@ public boolean haveTime()
         {
             return this.haveTime.getAsBoolean();
         }
+        
+        /**
+         * @return Returns the {@link MinecraftServer} that is currently ticking.
+         */
```

Comment:
```suggestion
        /**
         * {@return the server instance}
         */
```

Reply:
Manually merged: 610a74eff95f16fc9a7bc3090622fe02c3115074 (I replaced the `server` bit with `{@link MinecraftServer}`.

Reply:
Technically, `{@link MinecraftServer}` would be redundant as the return type already specifies that information.

Reply:
I agree that this is a bit redundant, though I think that one should be able to easily understand a method just by taking a glance at the Javadoc so that you don't have to look at the method signature as well. The Javadoc of the JDK does seem to contain this type of redundancy too a few times (e.g. `Integer#describeConstable`).

At the end though, it is up to you to decide whether I should keep the redundant link or not, I'll happily change the Javadoc if you wish so.

Reply:
My preference is to leave out the redundant link, since the method signature is always visible alongside the javadoc (whether in web, IntelliJ IDEA, or Eclipse) and the developer is likely already aware of the type of "the server instance" in this event.

Reply:
Alright, I've updated the Javadoc. It should now line up with the change you've suggested above. e585195ce6a30423d4a6d03fe996b1a6717a95b2